docs/src/using-turing/compiler.md

```diff
@@ -207,8 +207,8 @@ function inner_function(vi::Turing.VarInfo, sampler::Turing.AbstractSampler, ctx
 
     ... # The code above is repeated for the other 2 variables, y and TV
 
-    # Reset `vi.logp`
-    vi.logp = 0
+    # Reset `vi.logp[]`
```

Comment:
Maybe use less code and just write something like
```suggestion
    # Reset the `logp` accumulator
```
It's not really relevant here that one can access it with `vi.logp[]`, it seems?